index.js

```diff
@@ -58,6 +58,44 @@ const getColor = color => {
   }
 }
 
+/*
+ * Returns a string indicating a `light` or `dark` text variation based on the
```

Comment:
This PR looks good!

This documentation also looks good, but it could be helpful to link to the original writeup Scott did in Github as well in case there's some minor detail that's missing here.